Classes/UT3Cicada.uc

```diff
@@ -240,20 +240,60 @@ defaultproperties
     //==========END==========
     VehicleNameString = "UT3 Cicada"
 
-    GroundSpeed=2000
+    GroundSpeed=1600
+    MaxRandForce=2.0
+    RandForceInterval=0.95
+    RollTorqueMax=60 //100
+    RollTorqueStrafeFactor=130 /100
```

Comment:
Clearly a typo, this would mean division...

Reply:
Actually no, some vehicles do have different damage rates and I had to go up and down to get the exact, within the same 6 damage in each game it has to be right?  I think the SPMA was one of the cases where it needed to be higher than normal, smoke on the other hand was pretty much always the same but fire wasn't not sure why.

Yeah my mistake on the / I'll fix it